ejercicio2-html-y-css/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <form>
+            <div class="form-header">
+                <h2 class="rounded">Login</h2>
+            </div>
+            <label>Email</label>
```

Comment:
Acá te falta el atributo `for` que conecta con el input https://guias.makeitreal.camp/docs/html-y-css/formularios#labels

Reply:
Listo, también implementé los atributos for faltantes en varios label. Fue un olvido mío ya que a los inputs si coloqué los id correspondientes.